app/cli/cmd/plugins.go

```diff
@@ -272,8 +270,7 @@ func newPluginInstallCmd() *cobra.Command {
 	}
 
 	// Common flags
-	cmd.Flags().StringVarP(&file, "file", "f", "", "URL of the plugin to download (required)")
-	cmd.Flags().StringVarP(&filename, "filename", "", "", "Custom filename to save the plugin as (optional)")
+	cmd.Flags().StringVarP(&file, "file", "f", "", "URL or path to the plugin to install (required)")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&file, "file", "f", "", "URL or path to the plugin to install")
```
you might want to leave the `requirement` behavior to be exposed by the tool itself.

Reply:
Great point, thank you!